main.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+
+def main():
+    """
+    Main function.
```

Comment:
Docstring is insufficient. Should mention more about what this function is doing.

Reply:
Programmer: Ok have fixed with latest commit thx